src/arch/riscv/kernel/boot.c

```diff
@@ -214,13 +214,15 @@ extern char trap_entry[];
 BOOT_CODE static void
 init_cpu(void)
 {
-    /* Write trap entry address to stvec */
-    write_csr_env(tvec, trap_entry);
```

Comment:
I realize that it does not matter whether we write the trap entry address before or after we `activate_kernel_vspace` but reordering this with no justification in a commit that claims to only "Save the trap address" is very poor style